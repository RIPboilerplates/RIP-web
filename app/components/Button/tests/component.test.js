import React from 'react'
import renderer from 'utils/renderers'
// import { shallow } from 'enzyme'

import Component from '../component'

let component
let props

beforeEach(() => {
  component = null
  props = {}
})

describe('Button Component', () => {
  describe('Snapshot Renders', () => {
    it('default', () => {
      component = renderer(<Component {...props} />).toJSON()
      expect(component).toMatchSnapshot()
    })
  })
})
