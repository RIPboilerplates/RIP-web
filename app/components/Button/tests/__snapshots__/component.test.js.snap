// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Snapshot Renders default 1`] = `
<button
  className="styles__Container-eZxbLb czehpr"
  onClick={undefined}
>
  <span>
    Log Out Action!
  </span>
</button>
`;
